src/twisted/internet/interfaces.py

```diff
@@ -1364,7 +1371,7 @@ class IReactorInThreads(Interface):
     @since: 15.4
     """
 
-    def callInThread(callable, *args, **kwargs):
+    def callInThread(f: Callable[..., Any], *args, **kwargs):
```

Comment:
This is still an interface change callable -> f

Reply:
I took this out